assembl/static2/js/app/utils/translations.js

```diff
@@ -1286,6 +1288,8 @@ const Translations = {
       export: {
         defaultAnnotation: "You can export all of the data by clicking on the export button",
         surveyAnnotation: "You can export all the survey module data by clicking on the Export button",
+        threadAnnotation: "You can export all the thread module data by clicking on the Export butotn",
+        multicolumnAnnotation: "You can export all the multicolumn data by clicking on the Export button",
```

Comment:
`multi-column` or `Multi Column` as there is no word like this in English.

Reply:
Oups, true :)